client/packages/host/src/Help/Help.tsx

```diff
@@ -1,20 +1,36 @@
 import {
   AppBarButtonsPortal,
+  BookIcon,
   Box,
   useInitialisationStatus,
 } from '@openmsupply-client/common/src';
 import React from 'react';
 import { AppVersion } from '../components';
 import { SiteInfo } from '../components/SiteInfo';
+import { UserGuide } from '../Admin/UserGuide';
+import { HelpSection } from './HelpSection';
 
 export const Help: React.FC = () => {
   const { data: initStatus } = useInitialisationStatus();
+
+  const [activeSection, setActiveSection] = React.useState<number | null>(null);
+
+  const toggleSection = (index: number) => () =>
+    setActiveSection(activeSection === index ? null : index);
+
   return (
     <Box flex={1} padding={4} sx={{ maxWidth: 800 }}>
       <div>
         <p>Hello</p>
       </div>
-
+      <HelpSection
```

Comment:
Argh apologies for my misunderstanding of the design on Friday - are you okay to update this, no longer an accordion, just the title and then the link? 🙏 

![image](https://github.com/user-attachments/assets/5609154f-4eae-4556-a840-98b9148e2d3c)

Reply:
Sure no worries!